SPECS/policycoreutils/policycoreutils.signatures.json

```diff
@@ -1,10 +1,10 @@
 {
  "Signatures": {
-  "selinux-3.2.tar.gz": "cda0b315ea88f93b1e7b6aa9f48d3b568a5506af1f8f401f5d05fa2ce8c7008f",
-  "selinux-autorelabel": "817911a36413e8cb13fd5107ed94314d3f49962befd00acd9e86038acfa31e73",
-  "selinux-autorelabel-generator.sh": "ec37c4ed3d2168d367d2bb07c44ea16df052e123908e83010d84aaa1bda1cede",
-  "selinux-autorelabel-mark.service": "c7707e97067ae344b99a9b59845a4c7335c9ff4e470723bf2a213a6757877707",
-  "selinux-autorelabel.service": "106ff381e311119d389fd6f9421330a408d3406fab0b990690c73739b02377ac",
-  "selinux-autorelabel.target": "c340133060c163caf9e22f0624306bd528768e89b3d5faf7285f23d2bada9485"
+    "selinux-autorelabel": "dc8f5dce298120900ceb40a4ed3f2af071b8f71018041e3cba618dccd213b721",
+    "selinux-autorelabel-generator.sh": "a2936d3fa33f1a8fcbf5f71b510dca82d159f31ff31fb60a35044a7968789e8a",
+    "selinux-autorelabel-mark.service": "781b8340d415dbdf87508df815fef783222f426e8cd6be1a516748968f166169",
+    "selinux-autorelabel.service": "8812d16f3797ac94dd0ba6185e7d2dcedf436d7d9d043f9c2899c7cdeed6cfef",
+    "selinux-autorelabel.target": "c65e4e9694c1f534b4eadf1f437293782d90b08afdc9123494505a3d9529ac98",
+    "selinux-3.6.tar.gz": "97a5f11a3b05e0a2f26166168b1d7cbab83d0841d4189c88a0c8864f2eafae2e"
```

Comment:
FYI: upstream packages separate source tars for individual components such as policycoreutils. May be worth using that instead, as we do for other packages (libselinux, libsepol, etc.) https://github.com/SELinuxProject/selinux/wiki/Releases